module/VuFind/src/VuFind/Content/AbstractSyndeticsFactory.php

```diff
@@ -81,9 +80,9 @@ public function __invoke(
             ? substr($requestedName, 0, strlen($requestedName) - 4) : $requestedName;
 
         return new $className(
-            isset($config->Syndetics->use_ssl) && $config->Syndetics->use_ssl,
+            isset($config['Syndetics']['use_ssl']) && $config['Syndetics']['use_ssl'],
```

Comment:
Could simplify to:

```suggestion
            $config['Syndetics']['use_ssl'] ?? false,
```

...I think that's a bit easier to read.